### PR TITLE
Bump the runtime 21.08 and new Irrlicht (irrlichtmt)

### PR DESCRIPTION
**Pull request by farribeiro**
Rework of #33

Bump the runtime to 21.08 and new irrlicht (irrlichtmt) in the minetest module and now need patch to missing anothers not related files

### CODE REVIEW

**Comment by farribeiro** [comment]
none of the new irrlicht works with version 5.4.1

**Comment by farribeiro** [comment]
Btw... This changes is ready to new releases(5.5.0 or later) just change irrlichtmt, luajit, minetest, minetest_game current archives and sha256 checksum

File: net.minetest.Minetest.yaml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 app-id: net.minetest.Minetest
 runtime: org.freedesktop.Platform
-runtime-version: "20.08"
+runtime-version: "21.08"
+default-branch: stable
```
Comment (hfiguiere) [inline net.minetest.Minetest.yaml:4]:
```suggestion
```

This is not needed and actually more than annoying when developing.

Comment (farribeiro) [reply net.minetest.Minetest.yaml:4]: I think is more verbose declaring the branch

**Comment by farribeiro** [comment]
I like to put x-data-checker, but not all branches have github api output

The yaml code to minetest maybe this

```yaml
        x-checker-data:
          type: "json"
          url: "https://api.github.com/repos/minetest/minetest/releases/latest"
          version-query: ".tag_name"
          url-query: ".assets[] | select(.name== $version + \".tar.gz\") | .browser_download_url"
```

**Comment by farribeiro** [comment]
@Calinou can you review the PR ?

**Pull request merged**